department-of-veterans-affairs/va.gov-cms
# VACMS-17940: Knowledge Base landing page redesign.

## Description

Closes #_17940_.

## Testing done
Local testing

## Screenshots
<img width="1700" alt="KB-top" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/166155028/4502ce13-a58b-4ba3-9895-6f22d371da63">

<img width="1701" alt="KB-bottom" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/166155028/f1198915-ddb3-4566-98b3-3b421b2a7f2a">

## QA steps

What needs to be checked to prove this works?
Will need to work with developer on inputing content in the correct order for knowledge base landing page (/help/)
What needs to be checked to prove it didn't break any related things?
Other KB pages other than the landing page.
What variations of circumstances (users, actions, values) need to be checked?

As drupal developer with content editing permissions
1. Make sure your content has this structure
   - [ ] ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/166155028/d62c4e4d-0be0-4de5-8f3c-d9d344ae2e71)
   - [ ]  Follow designs for content. 

Page Title:
CMS Knowledge Base

Page introduction:
Learn about new features, find step-by-step instructions, and how-to guides for posting content on VA.gov website.

*NOTE FOR List of Links - make sure to add another item where there are multiple in a column. The "Final Link" prints out differently.

1. List of links
#Section Header
CMS Basics - Getting Started

#List of Links Description
Find step-by-step instructions on how to complete common tasks within Drupal.

#List of links 1st column
/help/cms-basics/how-to-find-content-to-edit
Hot to find content to edit
How to add a link to a file
/help/cms-basics/file-link

#List of links 2nd column
/help/video-link
How to link to video/audio
/help/cms-basics/how-to-view-revisions
How to view Revisions

#List of links 3rd column
/help/cms-basics/how-to-fix-broken-links-and-redirects
How to fix broken links and redirects
/help/how-to-upload-images
How to upload images

2. List of links
#Section Header
Account Support

#List of Links Description
Find additional information about CMS Accounts.

#List of links 1st column
/help/cms-basics/about-cms-account-permissions
About CMS account permissions
/help/cms-account-admin-policies
CMS account admin policies

#List of links 2nd column
/help/cms-account-administration
CMS account administration
/help/search?keyword=new+account#:~:text=Editorial%20workflow%20roles%20%E2%80%A6-,How%20to%20request%20a%20CMS%20account,-Request%20a%20Drupal
How to request a CMS account

#List of links 3rd column
/help/cms-basics/how-to-log-inout
How to log in/out
/help/cms-basics/blocked-accounts
Blocked accounts

3. List of links
#Section Header
All-in-One Editor Guides

#List of Links Description
A central link with a collection of articles related to specific editor workflows.

#List of links 1st column
/help/vet-centers
Vet Centers Editor Guide

#List of links 2nd column
/help/vamc
VA Medical Centers Editor Guide

#List of links 3rd column
help/veterans-benefits-administration-vba
VBA Editor Guide

4. Rich text
```
<h3 id="training">
    Find more help
</h3>
<p>
    Contact the help desk or register for training.
</p>
<p>
    <a class="usa-button" href="https://va-gov.atlassian.net/servicedesk/customer/portal/3/group/8/create/26">Contact Help Desk</a>
</p>
<p>
    <a class="usa-button" href="/node/6884" data-entity-type="node" data-entity-uuid="8b944ecc-ff8b-4254-ac37-385c12018b1d" data-entity-substitution="canonical">Training Sign-Up</a>
</p>
```

5. Rich text
```
<h3>
    About
</h3>
<p>
    <a href="/node/28393" data-entity-type="node" data-entity-uuid="2aa8cab6-d4c8-463f-85e6-897e83f52625" data-entity-substitution="canonical">About the CMS Knowledge Base</a>
</p>
```

### Definition of Done
The /help/ page looks like these designs: https://www.figma.com/design/KDbDZdukzNBLjObpjtMq2b/CMS-Knowledge-Base?node-id=274-1191&t=cKmCSYB9QT1Hl7d5-0

- [] Acceptance Criteria in related issue are met.

### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner

- [] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
